libs/core/kiln_ai/datamodel/test_attachment.py

```
@@ -0,0 +1,353 @@
+def test_base_kiln_file(tmp_path) -> Path:
+    test_file_path = tmp_path / "test_model.json"
+    data = {"v": 1, "model_type": "kiln_base_model"}
+
+    with open(test_file_path, "w") as file:
+        json.dump(data, file, indent=4)
+
+    return test_file_path
+
+
+@pytest.fixture
+def test_media_file(tmp_path) -> Path:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)